templates/chartmuseum/chartmuseum-dpl.yaml

```diff
@@ -70,17 +80,42 @@ spec:
           - name: NO_PROXY
             value: "{{ template "harbor.noProxy" . }}"
           {{- end }}
+          {{- if .Values.internalTLS.enabled }}
```

Comment:
What are these environment variables used for?

Reply:
It was used to let chartmuseum run in https mode.